examples/social-network/config/database.js

```diff
@@ -0,0 +1,31 @@
+const development = {
+  username: 'root',
+  password: 'root',
+  database: 'social_network_dev',
+  host: '127.0.0.1',
+  port: '3306',
+  dialect: 'mysql',
+  logging: false
+};
+
+const test = {
+  username: 'root',
+  password: 'root',
+  database: 'social_network_test',
+  host: '127.0.0.1',
+  port: '3306',
+  dialect: 'mysql',
+  logging: false
+};
+
+const production = {
+  username: 'root',
+  password: 'root',
+  database: 'social_network_prod',
+  host: '127.0.0.1',
+  port: '3306',
+  dialect: 'mysql',
+  logging: false
+};
+
+export { development, test, production };
```

Comment:
This looks great. One suggestion though.

For consistency with the previous structure of the json config file, we should `export default` and object literal of the config.

```javascript
export default {
  development: {
    /* ... */
  },

  test: {
    /* ... */
  },

  production: {
    /* ... */
  }
};
```